.github/workflows/cclf-import-test-deploy.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           CGO_ENABLED: 0
         run: |
           go build -o bin/bootstrap ./lambda/cclf/main.go
-          zip -j function.zip bin/bootstrap
+          zip -j function.zip bin/bootstrap ../conf/configs/dev.yml ../conf/configs/test.yml ../conf/configs/prod.yml
```

Comment:
Sandbox missing?  Also can we do `../conf/configs/*.yml`?

Reply:
the only env that we don't do attribution in is sandbox, so this is deliberate! I can make the changes for the wildcard if we are OK with having sandbox in there but not needed. Thoughts?